portal/src/main/webapp/js/src/oncoprint/webgl/dist/oncoprint-bundle.js

```diff
@@ -1656,14 +1657,17 @@ var OncoprintLabelView = (function () {
 		} else {
 		    var hovered_track = isMouseOnLabel(view, evt.pageY - view.$canvas.offset().top);
 		    if (hovered_track !== null) {
-			var $tooltip_div = $('<div>');
+			var $tooltip_div = $('<section>');
```

Comment:
Considering what block element the boldfaced label should actually be semantically, I identified it as a heading of the tooltip, whatever the rank of that heading would be in the structure of the content (I guessed about 5). I can leave this commit out though, as the concatenation of label and description is no longer a problem now that I made the description itself a block element. Besides, the tooltip would have been an `<aside>` to the flow of content if anything.